core/jpa-support/src/main/java/io/thorntail/jpa/impl/JpaMessages.java

```diff
@@ -34,7 +34,4 @@
     @LogMessage(level = Logger.Level.INFO)
     @Message(id = 11 + JPA_OFFSET, value = "initializing persistence contexts")
     void persistencecontextInitialization();
-
-    @Message(id = 12 + JPA_OFFSET, value = "%s when attempting to wrap EntityManager with FullTextEntityManager")
-    RuntimeException errorWrappingEntityManagerForSearch(Exception e);
```

Comment:
Probably doesn't matter now, but I'd consider removing log messages dangerous, as it will undoubtedly cause log ID reuse, which is wrong.

Reply:
The log message being removed was used by the old reflection-based code I wrote. I wouldn't pull it if we had done a release with it. :+1: